graph-manager-docs/source/slack-integration.md

```diff
@@ -3,68 +3,75 @@ title: Integrating Graph Manager with Slack
+After you [connect at least one Slack channel](#connecting-a-slack-channel), you can set up each of the individual notification types from the other sections of your graph's Integrations page:
 
-## Notifications
+<img src="./img/integrations/integrations-tab.jpg" class="screenshot" alt="The integrations page in Graph Manager"></img>
 
-In Graph Manager, you can configure notifications that are triggered on the performance data of your graph, like error percentages and request latencies. This is particularly useful for detecting anomalies, especially around releases. Notifications can be configured to monitor the following metrics for either your entire GraphQL service or individual operations:
+## Daily reports
 
-- **Request rate:**  requests per minute
-- **Request duration:** p50/p95/p99 service time
-- **Error rate:** errors per minute
-- **Error percentage:** the number of requests with errors, divided by total
-  requests
+Graph Manager sends your graph's daily report around 9am in your specified time zone. The report provides a high-level summary of what your GraphQL API delivered in the previous day:
 
-The triggers you set up are evaluated on a rolling five minute window. For example, you can configure a notification to trigger when an operation's error rate exceeds 5%. In production, if 6 out of 100 requests result in an error during the last five minutes, the alert will trigger with an error rate of 6%. Once the error rate falls back below 5% your notification will resolve. Here's an example of what the notification looks like:
+<img src="./img/integrations/slack-report.png" alt="Slack daily report" class="screenshot"></img>
 
-![Slack Alert](./img/integrations/slack-notification.png)
+### Acting on report data
+
+The daily report provides an actionable summary of what's happened in your API over the last 24 hours. Here’s how you can use it to identify issues:
 
-## Schema Notifications
+*  **Request rate:** This shows you how many queries are hitting your server every minute, along with a list of the most popular operations. If you see a significant dip in this value, it might mean that queries aren’t able to reach your server, or that a particular client is down.
+*  **p95 service time:** An operation's p95 response time indicates that 95% of that operation's executions complete _faster_ than the reported value. You can use this to identify that your API is overloaded and users are seeing long loading delays, or to find out which queries are taking the longest to run. This is often connected to UI performance, so a 500ms query probably means some part of your UI is taking that long to display.
+*  **Error percentage:** This shows you how many of your GraphQL requests produce an error result. Spikes in errors might be the result of an underlying back-end malfunction. You can also see which of your operations are most error-prone.
```

Comment:
Should we also mention the list of operations with the highest error percentage? Curious why it was included in the first bullet point but not the subsequent.